Classes/Command/ExtensionCommandController.php

```diff
@@ -161,6 +161,33 @@ public function dumpAutoloadCommand()
     }
 
     /**
+     * List active extensions
+     *
+     * @param bool $raw True for a comma separated list of extension keys
+     * @return void
+     */
+    public function listActiveCommand($raw = false)
+    {
+        $extensionInformation = [];
+        foreach ($this->packageManager->getActivePackages() as $package) {
+            $metaData = $package->getPackageMetaData();
+            $extensionInformation[] = [
+                'package_key' => $package->getPackageKey(),
+                'version' => $metaData->getVersion(),
+                'description' => $metaData->getDescription(),
+            ];
+        }
+        if ($raw) {
+            $this->outputLine('%s', [implode(',', array_column($extensionInformation, 'package_key'))]);
```

Comment:
isn't it easier for later processing on command line if these are separated by line feed?